worldedit_commands/safe.lua

```diff
@@ -1,3 +1,9 @@
+local area_protection = {}
+
+if minetest.get_modpath("areas") then
+    area_protection.areas = areas
+end

```

Comment:
It seems rather odd to create a table just to store a reference to the areas API.  I think it would be better to just return the areas API table or `nil`.  If support for another protection system is ever added you could have an API that abstracts over the backend APIs.